src/components/CommentsCard.jsx

```diff
@@ -1,7 +1,42 @@
+import { fetchCommentsByArticle } from "../../utils/api";
+import { useState, useEffect } from "react";
+import { useParams } from "react-router-dom";
+
 export default function CommentsCard() {
+  const { article_id } = useParams();
+  const [comments, setComments] = useState();
+  const params = useParams();
+
+  useEffect(() => {
+    fetchCommentsByArticle(article_id).then((commentsData) => {
+      // console.log(typeof commentsData.comments);
+      setComments(commentsData.comments);
+    });
+  }, []);
+
+  console.log(params, "params");
+  console.log(comments, "comments");
+
+  if (!comments) {
+    return <div>Comment section loading, please wait.</div>;
+  }
+
   return (
     <>
       <h3>Comments Section</h3>
+
+      {comments.map((comment) => (
+        <div key={comment.comment_id}>
+          <h5>{comment.author}</h5>
+          <p>{comment.body}</p>
+          <p>{comment.created_at}</p>
```

Comment:
Currently this line of code comes out as a not very friendly string of `2020-06-16T07:07:00.000Z` for example. It also doesn't currently tell me what it represents? it might be worth tweaking this a bit and making it more obvious that this was a comment posted at 'X'. Although turning this date/time string into a nicer format isn't critical, it would be a nice touch.